Encountered `undefined: genstatic.AssetInfo` when trying to build traefik v2

### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?
I'd like to build traefik from source, I've done the following:

- Clone the traefik repository
- Switch to branch `v2.0.4`
- Run `go generate`
- Run `go build ./cmd/traefik/traefik.go`



### What did you expect to see?
Successfully built.


### What did you see instead?
I got an error: `cmd/traefik/traefik.go:99:97: undefined: genstatic.AssetInfo`


### Output of `traefik version`: (_What version of Traefik are you using?_)
v2.0.4

### What is your environment & configuration (arguments, toml, provider, platform, ...)?
Mac OSX, just build, no configuration toml.

--- a/docs/content/contributing/building-testing.md
+++ b/docs/content/contributing/building-testing.md
@@ -62,6 +62,7 @@ Requirements:
 
 - `go` v1.13+
 - environment variable `GO111MODULE=on`
+- go-bindata `GO111MODULE=off go get -u github.com/containous/go-bindata/...`
 
 !!! tip "Source Directory"
 

